src/ontology/mondo-edit.obo

```diff
@@ -251721,11 +252060,13 @@ subset: ordo_disorder {source="Orphanet:85275"}
 subset: ordo_malformation_syndrome {source="Orphanet:85275"}
 subset: orphanet_rare {source="Orphanet:85275"}
 subset: rare
+synonym: "anop1" RELATED [OMIM:309800]
 synonym: "ANOP1, formerly" EXACT [OMIM:301590]
```

Comment:
While this PR generally shows the correct representation of synonyms from OMIM that have the string ", formerly", this example (`synonym: "ANOP1, formerly" EXACT [OMIM:301590]`) is not correct. 

@joeflack4 why is this one not represented as `synonym: "ANOP1" RELATED [OMIM:301590]`? And why are related synonyms still tagged with owl:deprecated from the OMIM Ingest pipeline?